src/Amazon.Lambda.Tools/LambdaDefinedCommandOptions.cs

```diff
@@ -417,5 +417,29 @@ public static class LambdaDefinedCommandOptions
                 ValueType = CommandOption.CommandOptionValueType.StringValue,
                 Description = "Docker image name and tag in the 'name:tag' format."
             };
+        public static readonly CommandOption ARGUMENT_EPHEMERAL_STORAGE_SIZE =
+            new CommandOption
+            {
+                Name = "Ephemerals Storage Size",
+                Switch = "--ephemerals-storage-size",
```

Comment:
Also should this match CLI parameter `--ephemeral-storage` like we do for all parameter ?

https://docs.aws.amazon.com/cli/latest/reference/lambda/create-function.html

Reply:
EphemeralStorage is an object so for the CLI when you use `--ephemeral-storage` you have to pass in JSON. This till doesn't have the concept of passing in JSON for properties nor do I want to make users make JSON for just setting the size. What we usually do in this tool is flatten the parameters to make it easier to set. If the is another property added to EphemeralStorage we would most likely add another top level switch.